testing/e2e.sh

```diff
@@ -542,6 +534,36 @@ function test_py_image_deps_as_layers() {
   EXPECT_CONTAINS "$(bazel run testdata/test:py_image_using_layers)" "Successfully imported six 1.11.0"
 }
 
+function test_dockerfile_image_basic() {
+  cd "${ROOT}/testing/examples/basic"
+  clear_docker
+  bazel test ...
+}
+
+function test_dockerfile_image_extended() {
+  cd "${ROOT}/testing/examples/extended"
+  clear_docker
+  bazel test ...
+}
+
+function test_dockerfile_image_java_app() {
+  cd "${ROOT}/testing/examples/java_app"
+  clear_docker
+  bazel test ...
+}
+
+function test_dockerfile_image_apt_pkgs() {
+  cd "${ROOT}/testing/examples/run_instruction_apt_pkgs"
+  clear_docker
+  bazel test --host_force_python=PY2 ...
+}
+
+function test_dockerfile_image_arbitrary() {
+  cd "${ROOT}/testing/examples/run_instruction_arbitrary"
+  clear_docker
+  bazel test --host_force_python=PY2 ...
```

Comment:
add .bazelrc files to the projects with this flag

Reply:
Done.
There is only one "internal" project for all examples, so added one such file.